scripts/jenkins/post-e2e-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
safer to use `/usr/bin/env bash`, as some distros (like ubuntu) use odd things like `dash` in this case which can cause this script to fail .. better to describe the environment explicitly here that you want to use